<launch>
<param name="param/mobilenet/image_src" type="string" value="/zed_node/left/image_rect_color" />
<!--<param name="param/mobilenet/image_src" type="string" value="/wheelchair_robot/left_camera/image_raw" />-->
<!--<param name="param/mobilenet/image_src" type="string" value="/still_image_publisher/image_raw" />-->
<param name="param/mobilenet/confidence_threshold" type="double" value="0.5" />

<node pkg="mobilenet" name="mobilenet" type="detect_and_transform.py" output="screen"></node>
<!--<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
<remap from="image" to="/wheelchair_robot/mobilenet/annotated_camera" />
</node>-->
</launch>
